### ISSUE TITLE
Adding a secondary org to an existing config crashes User Sync

### ISSUE THREAD

**Issue by adobeDan** [adobe-apiplatform/user-sync.py#293]
A customer with an existing (working) User Sync configuration has added a second org on the Adobe side, and wants to push some of the existing users from the first org into the second org (and assign them to groups).  They are running into an issue, because User Sync assumes that any users in the primary have already been pushed to the secondary, so it's crashing with the message:
```log
2017-10-29 00:01:11 4513 CRITICAL processor - Shouldn't happen! In secondary umapi stable_2, the following users were not found: [u'federatedID,mr.ed@talkinghorses.tv,']
```

**Comment by phil-levy** [comment]
Not sure what this is about.  You should NEVER create a user in a secondary org.  UMAPI handles adding the user to the secondary org if that user is added to a group in the secondary org.  No other processing should be necessary.  It is an error to attempt to create the user in the secondary org because the user already exists.

**Comment by adobeDan** [comment]
@phil-levy Your understanding of the bug is incorrect.  The problem is that User Sync was assuming that any users who have to be pushed to a secondary org in order to be assigned a group there have already been created in the secondary org.  But, of course, that won't be true unless the secondary org was already known when the user was created.  For this customer (and many others to come) who buy Stock after they've provisioned their users, this won't be true, because every Stock PAYG purchase is a new contract which creates a new organization with a new console.  So the customer has to use UMAPI to push their existing users from their existing console into the new console's groups so they can use Stock.  And that caused a crash in User Sync, because it wasn't prepared to push existing users to the new org.

I'm not sure why you reopened this, since it was closed fixed in the posted v2.2.2rc2 build, and I've tested the change extensively.  I'm re-closing this.
